apps/lumberjack-app/src/app/forest.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable, OnDestroy } from '@angular/core';
+import { ReplaySubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ForestService implements OnDestroy {
+  #fire$ = new ReplaySubject<void>(1);
+
+  fire$ = this.#fire$.asObservable();
+
+  constructor() {
+    setTimeout(() => this.#fire$.next(), 2000);
+  }
+
+  ngOnDestroy(): void {
+    this.#fire$.complete();
+  }
```

Comment:
Why not a timer?

```suggestion
fire$ = timer(2000).pipe(mapTo(true));
```

Reply:
That is a cold observable.

Reply:
I'm ok with the subject btw, but it just seems unnecessarily verbose.

Reply:
Blame that on Angular not including native declarative state management for side effects 😄